spec/models/review_spec.rb

```diff
@@ -31,7 +38,67 @@
         review = Review.new(card_id: card.id, answer: "wey")
         expect(review.check_translation).to be_falsey
       end
+
+      it "increase failed_review_count by one" do
+        review = Review.new(card_id: card.id, answer: "wey")
+        expect {
+          review.check_translation
+        }.to change{card.reload.failed_review_count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.